src/plugins/ecs/__init__.py

```diff
@@ -479,5 +527,81 @@ def tasks_get_lookup_term(text):
                 return data[(data.lower().find('---') + 3):]
 
 
-
-
+#retrieve json data from super jenkins for build urls
+def get_superjenkins_data(superjenkins_link,beginning_script_tag,ending_script_tag):
+
+    # get json to get individual services/application build links
+    cached_items = ""
+    cached_array = ""
+
+    try:
+        returned_data = requests.get(superjenkins_link)
+        returned_data_iterator = returned_data.iter_lines()
+
+        for items in returned_data_iterator:
+            if beginning_script_tag in items:
+                cached_items = items.replace(beginning_script_tag, "").replace(ending_script_tag, "")
+                break
+
+        for items in json.loads(cached_items):
+            cached_array = json.loads(cached_items)[items]
+
+    except Exception, e:
+        print "Error in retrieving and creating json for build urls ==> " + str(e)
+
+    return cached_array
+
+
+#retrieve data from config files for compare
+def get_in_ecs_compare_data(config, args, args_eval):
+
+	result = dict()
+
+	if args_eval == 3:
+		result['cluster_name'] = None
+		result['region_name'] = args[1]
+		result['account'] = args[2]
+	elif args_eval == 4:
+		result['cluster_name'] = args[0]
+		result['region_name'] = args[2]
+		result['account'] = args[3]
+
```

Comment:
I'm always a fan of comments in code. You could say what you're doing in the following for loop - what you're trying to get. I'm not completely clear on what you're trying to do here. Better variable/parameter names might help.